weave/tests/test_index_pandas.py

```diff
@@ -100,6 +101,33 @@ def test_sync_index_calls_generate_index_if_no_index(test_pantry):
     )
     assert len(pantry.index.to_pandas_df()) == 1
 
+def test_sync_index_autocleanup_if_false(test_pantry):
+    """Tests that IndexPandas.sync raises the correct warning with
+    auto-cleanup turned off."""
+    pantry = Pantry(
+        IndexPandas,
+        pantry_path=test_pantry.pantry_path,
+        file_system=test_pantry.file_system,
+        sync=True,
+        auto_cleanup=False,
+    )
+    pantry.index.to_pandas_df()
```

Comment:
why are we calling to_pandas_df() here?

Reply:
its not needed I think I just forgot to remove it when I was moving code over from my demo

Reply:
it has been removed